jobs/atc/templates/atc_ctl.erb

```diff
@@ -185,6 +186,7 @@ case $1 in
       <% end %> \
       --generic-oauth-token-url <%= esc(p("generic_oauth.token_url")) %> \
       --generic-oauth-display-name <%= esc(p("generic_oauth.display_name")) %> \
+      --generic-oauth-ca-cert $GENERIC_OAUTH_CA_CERT \
```

Comment:
this should be passed conditionally since if it's not set it'll be pointing to an empty file, which isn't valid